src/systems/eu/eu-component/eu-component-profile-topbar/eu-component-profile-topbar.twig

```diff
@@ -1,44 +1,57 @@
 {#
-  Component interface:
-    - "context_nav" (object) - data for @ecl/eu-component-context-nav component composition
-    - "profile" (object): data contains profile-specific information
-      - "image" (object): data for @ecl/eu-style-image component composition
+  Parameters:
+    - "context_nav" (object) - data for @ecl/generic-context-navs component composition
```

Comment:
This one should be `generic-component-context-nav` here and in the implementation in the system implementation.